### PR TITLE
RDoc-1531: Added page How to Configure Revisions

### PR DESCRIPTION
**Pull request by morhilai**
- created new folder Client-API/Operations/Revisions
- c# and java versions
- fixed a table in Server/.../Revisions page

### CODE REVIEW

File: Documentation/4.1/Raven.Documentation.Pages/client-api/operations/revisions/configure-revisions.dotnet.markdown [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+| - | - | - |
+| **MinimumRevisionsToKeep** | The minimum number of revisions to keep per document | `null` |
+| **MinimumRevisionAgeToKeep** | The minimum amount of time to keep each revision. [Format of `TimeSpan`](https://docs.microsoft.com/en-us/dotnet/api/system.timespan). | `null` |
+| **Disabled** | Indicates whether to completely disable revisions for documents in this collection | `false` |
+| **PurgeOnDelete** | When a document is deleted, this indicates whether all of its revisions should be deleted as well | `false` |
+
+A revision is only deleted if both the `MinimumRevisionsToKeep` for that document is exceeded, **and** the revision is older 
+than the `MinimumRevisionAgeToKeep` limit. The oldest revisions are deleted first.  
+
+* By default both these options are set to `null` which has the same effect as setting them both to `0`: no revisions will
```
Comment (maximburyak) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/operations/revisions/configure-revisions.dotnet.markdown]: Null means there is no limitation

File: Documentation/4.1/Raven.Documentation.Pages/client-api/operations/revisions/configure-revisions.dotnet.markdown [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        public RevisionsCollectionConfiguration Default;
+}
+{CODE-BLOCK/}
+
+| Property | Description | Default |
+| - | - | - |
+| **Collections** | A dictionary in which the keys are collection names, and the values are the corresponding configurations | `null` |
+| **Default** | An optional default configuration that applies to any collection not listed in `Collections` | `null` |
+
+If a collection is not listed in `Collections`, and `Default` has not been set, the default values listed in the table
```
Comment (maximburyak) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/operations/revisions/configure-revisions.dotnet.markdown]:
```suggestion
If a collection is not listed in `Collections` and `Default` has not been set, the default values listed in the table 
```

File: Documentation/4.1/Raven.Documentation.Pages/client-api/operations/revisions/configure-revisions.dotnet.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/operations/revisions/configure-revisions.dotnet.markdown]: Fixed

File: Documentation/4.1/Raven.Documentation.Pages/client-api/operations/revisions/configure-revisions.dotnet.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/operations/revisions/configure-revisions.dotnet.markdown]: Fixed

**Pull request merged**